lib/stripe_mock/test_strategies/base.rb

```diff
@@ -21,6 +21,37 @@ def generate_card_token(card_params={})
         stripe_token.id
       end
 
+      def create_coupon_params(params = {})
+        {
+          id: '10BUCKS',
+          amount_off: 1000,
+          currency: 'usd',
+          max_redemptions: 100,
+          metadata: {
+            created_by: 'admin_acct_1'
+          },
+          duration: 'once'
+        }.merge(params)
+      end
+
+      def create_coupon_percent_of_params(params = {})
```

Comment:
Sorry, I wasn't specific. I do see that `create_coupon_params` gets used, but I couldn't find where `create_coupon_percent_of_params` got used / tested. Maybe just forgot to write a test for it?